### PR TITLE
Move types from i3s-converter to particular modules

### PR DESCRIPTION
**Pull request by dryabinin-actionengine**

### CODE REVIEW

File: modules/gltf/src/lib/types/gltf-types.ts [hunk 1]
```diff
@@ -54,6 +54,14 @@ export type GLTF_KHR_draco_mesh_compression = {
   extras?: any;
 };
 
+export type GLTFHeader = {
```
Comment (ibgreen) [inline modules/gltf/src/lib/types/gltf-types.ts:57]: Nit: Naming - This is not part of glTF spec, it is part of the 3DTILES spec. Isn't this in fact `BatchTable` above?

File: modules/i3s/src/index.ts [hunk 2]
```diff
@@ -1,3 +1,24 @@
 export {I3SLoader} from './i3s-loader';
 export {I3SContentLoader} from './i3s-content-loader';
 export {I3SAttributeLoader, loadFeatureAttributes} from './i3s-attribute-loader';
+export {
```
Comment (ibgreen) [inline modules/i3s/src/index.ts:4]: `export type`

File: modules/gltf/src/index.ts [hunk 3]
```diff
@@ -29,3 +29,5 @@ export {GLBWriter} from './glb-writer';
 // glTF Data Access Helper Class
 export {default as GLTFScenegraph} from './lib/api/gltf-scenegraph';
 export {postProcessGLTF} from './lib/api/post-process-gltf';
+export {Mesh} from './lib/types/gltf-json-schema';
+export {GLTFObject, GLTFHeader} from './lib/types/gltf-types';
```
Comment (ibgreen) [inline modules/gltf/src/index.ts:33]: `export type`

File: modules/i3s/src/types.ts [hunk 4]
```diff
@@ -1,3 +1,272 @@
+import type {GLTFMaterial} from '@loaders.gl/gltf';
+import {Quaternion, Vector3} from '@math.gl/core';
+import {Mesh} from '@loaders.gl/gltf';
```
Comment (ibgreen) [inline modules/i3s/src/types.ts:3]: `import type`, and add the dependencies per the eslint-warnings below

File: modules/3d-tiles/src/types.ts [hunk 5]
```diff
@@ -0,0 +1,28 @@
+import type {GLTFObject, GLTFHeader} from '@loaders.gl/gltf';
+import {Matrix4, Vector3} from '@math.gl/core';
+
+export type TableJson = {
```
Comment (ibgreen) [inline modules/3d-tiles/src/types.ts:4]: Nit: `BatchTableJson`? `BatchTableData`

**Review by ibgreen — APPROVED** [review]

File: modules/gltf/src/lib/types/gltf-types.ts (hunk 1 above)
Comment (dryabinin-actionengine) [reply modules/gltf/src/lib/types/gltf-types.ts]: Moved to 3dTiles type file.

File: modules/i3s/src/index.ts (hunk 2 above)
Comment (dryabinin-actionengine) [reply modules/i3s/src/index.ts:4]: done

File: modules/gltf/src/index.ts (hunk 3 above)
Comment (dryabinin-actionengine) [reply modules/gltf/src/index.ts]: done

File: modules/i3s/src/types.ts (hunk 4 above)
Comment (dryabinin-actionengine) [reply modules/i3s/src/types.ts]: Done. I don't have any eslint warnings below.

File: modules/3d-tiles/src/types.ts (hunk 5 above)
Comment (dryabinin-actionengine) [reply modules/3d-tiles/src/types.ts]: Renamed to `BatchTableJson`

**Pull request merged**